pkg/route/api/v1/types.go

```diff
@@ -106,4 +114,11 @@ const (
 	TLSTerminationPassthrough TLSTerminationType = "passthrough"
 	// TLSTerminationReencrypt terminate encryption at the edge router and re-encrypt it with a new certificate supplied by the destination
 	TLSTerminationReencrypt TLSTerminationType = "reencrypt"
+
+	// TLSInsecureDisable disables insecure connections for an edge-terminated route.
+	TLSInsecureDisable TLSInsecureType = "disable"
+	// TLSInsecureExpose allows insecure connections for an edge-terminated route.
+	TLSInsecureExpose TLSInsecureType = "expose"
+	// TLSInsecureRedirect redirects insecure connections for an edge-terminated route.
```

Comment:
Expand the description to give an example - for instance, "Routers that support HTTP will redirect the connection to HTTPS"